docs/src/main/sphinx/release/release-429.md

```diff
@@ -0,0 +1,51 @@
+# Release 429 (11 Oct 2023)
+
+## General
+
+* Allow {doc}`/sql/show-functions` for a specific schema. ({issue}`19243`)
+* Add security for function listing. ({issue}`19243`)
+
+## Security
+
+* Stop performing security checks for functions in the `system.builtin` schema. ({issue}`19160`)
+* Remove support for using function kind as a rule in file-based access control. ({issue}`19160`)
+
+## Web UI
+
+* Log out from a Trino OAuth session when logging out from the Web UI. ({issue}`13060`)
+
+## Delta Lake connector
+
+* Allow using the `#` and `?` characters in S3 location paths or URLs. ({issue}`19296`)
+
+## Hive connector
+
+* Add support for `varchar` to `date` coercion. ({issue}`19201`)
+* Add support for `decimal` to `tinyint`, `smallint`, `integer`, and `bigint`
+  coercions for partitioned tables. ({issue}`19201`)
```

Comment:
"Coercions" is the wrong term here. Is this about supporting partitions where the declared type of a column in the partition doesn't match the type in the table? Or for evolving the schema of a table? 

It seems similar to the entry in release 427: "Allow changing a column’s type from double to varchar in Hive tables."